.github/renovate.json

```diff
@@ -0,0 +1,8 @@
+{
+  "automerge": true,
+  "enabledManagers": ["github-actions", "npm"],
+  "labels": ["dependencies", "javascript"],
+  "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
+  "stabilityDays": 3,
+  "ignoreDeps": ["cypress"]
+}
```

Comment:
This is very nitpicky and feel free to ignore if you hate it, but... what are your thoughts on keeping JSON config file keys in sorted alphabetical order? Personally I like it to make them easier to read through and would request it here, but I won't block the PR on it. 😄

Reply:
I like that too, I kept it unsorted cause it was liike that in the suggested PR😅